### PR TITLE
Updated near and far descriptions for Orthographic and Perspective Camera

### PR DESCRIPTION
**Pull request by looeee**
Added note on expected ranges for camera near and far propertes

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

File: docs/api/cameras/OrthographicCamera.html [hunk 1]
```diff
@@ -64,7 +64,12 @@
 		<div>Camera frustum bottom plane.</div>
 
 		<h3>[property:Float far]</h3>
-		<div>Camera frustum far plane. Default is *2000*.</div>
+		<div>
+		Camera frustum far plane. Default is *2000*.<br /><br />
+
+		Note that Three does not currently give warnings for incorrect values here, however the expected
```
Comment (WestLangley) [inline docs/api/cameras/OrthographicCamera.html]: I think we use "three.js", not "Three". It is not capitalized. There are multiple occurrences of this error.

File: docs/api/cameras/OrthographicCamera.html [hunk 2]
```diff
@@ -64,7 +64,12 @@
...
+		range is between the current value of the [page:.far far] plane and infinity.
```
Comment (WestLangley) [inline docs/api/cameras/OrthographicCamera.html]: current value of the `near` plane. Multiple occurrences.

File: docs/api/cameras/PerspectiveCamera.html [hunk 3]
```diff
@@ -80,7 +85,12 @@
 
 
 		<h3>[property:Float near]</h3>
-		<div>Camera frustum near plane. Default is *0.1*.</div>
+		<div>
+			Camera fustum near plane. Default is *0.1*.<br /><br />
+
+			Note that Three does not currently give warnings for incorrect values here, however the expected
+			range is between 0 and the current value of the [page:.far far] plane.
```
Comment (WestLangley) [inline docs/api/cameras/PerspectiveCamera.html]: 0 is not a valid value for perspective camera.

File: docs/api/cameras/OrthographicCamera.html (hunk 1 above)
Comment (looeee) [inline docs/api/cameras/OrthographicCamera.html]: OK, I'll update this throughout the docs. Should it be capitalised at the start of sentences?

File: docs/api/cameras/PerspectiveCamera.html (hunk 3 above)
Comment (looeee) [inline docs/api/cameras/PerspectiveCamera.html]: Is 0 a valid value for the near plane of the orthographic camera?

Comment (WestLangley) [inline docs/api/cameras/PerspectiveCamera.html]: Yes.

File: docs/api/cameras/OrthographicCamera.html (hunk 1 above)
Comment (WestLangley) [reply docs/api/cameras/OrthographicCamera.html]: @looeee A bit late, but, no.  :)

**Pull request merged**